Traceback (most recent call last):
  File "/home/ubuntu/multipass/morning-problem/n_sum-TecNicShan/soln/n_sum.py", line 7, in <module>
    print(" ".join(x))
TypeError: sequence item 0: expected str instance, int found

Expected file "test-cases/as-274-21-n_sum.py-test/Expected/002-stdout.txt" has no output file match
Expected file "test-cases/as-274-21-n_sum.py-test/Expected/002-stderr.txt" has no output file match
